tffm/models.py

```diff
@@ -27,17 +24,16 @@ class TFFMClassifier(TFFMBaseModel):
     def __init__(self, **init_params):
         assert 'loss_function' not in init_params
         use_weights = init_params.get('use_weights', False)
+        init_params['model_type'] = 'classifier'
         if use_weights:
             init_params['loss_function'] = loss_weighted_logistic
         else:
             init_params['loss_function'] = loss_logistic
-    
+
         self.init_basemodel(**init_params)
 
     def preprocess_target(self, y_):
```

Comment:
can remove `preprocess_target` everywhere since it is no longer used. Use also in `TFFMRegressor` and
here:
https://github.com/aloisg/tffm/blob/6325da627e323a441d827f6bc00412c2235e2312/tffm/base.py#L197
Might also pop up somewhere else